frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```
@@ -153,22 +152,23 @@
   "Add a parallel task": "Add a parallel task",
   "Task does not exist": "Task does not exist",
   "No Tasks": "No Tasks",
-  "Select task": "Select task",
+  "Select Task": "Select Task",
   "Delete Task": "Delete Task",
-  "Pipeline Runs": "Pipeline Runs",
-  "Select a project to view the list of {{pipelinesPageTitle}}": "Select a project to view the list of {{pipelinesPageTitle}}",
+  "Pipeline runs": "Pipeline runs",
```

🤔  Pipeline Run is a Custom Resource... capitalize?
```suggestion
  "Pipeline Runs": "Pipeline Runs",
```

---

If it's a custom resource capitalizing sounds good to me.